documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -422,40 +421,44 @@ and can be configured using the `jvmOptions` property in following resources:
 * `KafkaMirrorMaker2.spec`
 * `KafkaBridge.spec`
 
-Only the following JVM options are supported:
+You can specify the following options in your configuration:
 
-`-Xms`:: Configures the minimum initial allocation heap size when the JVM starts.
-`-Xmx`:: Configures the maximum heap size.
+`-Xms`:: Minimum initial allocation heap size when the JVM starts.
+`-Xmx`:: Maximum heap size.
+`-XX`:: Advanced runtime options for the JVM.
+`gcLoggingEnabled`:: xref:con-common-configuration-garbage-collection-{context}[Enables garbage collector logging].
+`javaSystemProperties`:: Additional system properties.
```

Comment:
We should have an example for this.

@sknot-rh How are they configured? is it just the property name and value and the `-D` is appended by Strimzi?

Reply:
I guess Standa didn't provided anything. But this is something what can be used as an example:

```yaml
    # ...
    jvmOptions:
      javaSystemProperties:
        - name: javax.net.debug
          value: ssl
    #...
```

Reply:
Checking w/ Stanislav

![image](https://user-images.githubusercontent.com/47596553/104724986-5f5ae480-5729-11eb-9529-1a1f93039281.png)